packages/commonwealth/server/controllers/server_threads_methods/get_threads_by_id.ts

```diff
@@ -103,6 +103,9 @@ export async function __getThreadsById(
           },
         ],
       },
+      {
+        model: this.models.ThreadVersionHistory,
```

Comment:
11 tables and 13 joins in this query :skull:

Reply:
Should we load version history here or should we have a separate v history query to show versions when the users asks for that view?

Reply:
It may be useful to have a separate route since the first step in transitioning content to blob storage is moving version history off Postgres so fetching that content in isolation would simplify that transition...

Reply:
The issue here is we need some identifier to show that a comment/thread has a version history, becuase the UI changes based on it